### PR TITLE
run in CI without using docker compose

### PR DESCRIPTION
**Pull request by briskt**

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -12,5 +12,10 @@ fi
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 "$DIR"/build.sh
 
+apt-get update && apt-get -y install curl
```
Comment (hobbitronics) [inline scripts/deploy.sh:15]: should ubuntu-latest already have curl?

**Review by hobbitronics — APPROVED** [review]
Looks good, just a few questions to consider.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -36,4 +35,4 @@ jobs:
           GOOGLE_AUTH_PRIVATE_KEY_ID: ${{ secrets.GOOGLE_AUTH_PRIVATE_KEY_ID }}
           GOOGLE_AUTH_PRIVATE_KEY: ${{ secrets.GOOGLE_AUTH_PRIVATE_KEY }}
           GOOGLE_AUTH_TOKEN_URI: ${{ secrets.GOOGLE_AUTH_TOKEN_URI }}
-        run: docker compose -f action-services.yml run deploy
+        run: ./scripts/deploy.sh ${{ vars.STAGE }}
```
[the remark this answers is not in the document]
Comment (briskt) [reply .github/workflows/ci.yml:38]: I don't think so. What drew your attention to that?

File: scripts/deploy.sh (hunk 1 above)
Comment (briskt) [reply scripts/deploy.sh:15]: Good call. I am so used to running in Docker that I forget that the ubuntu-latest runner has a lot of tools already installed.

File: .github/workflows/ci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (hobbitronics) [reply .github/workflows/ci.yml:38]: I probably should have asked in the script itself, but maybe ai hallucinated it and I wasn't sure if runners had root access, but I've never seen it used in an action.

[the remark this answers is not in the document]
Comment (briskt) [reply .github/workflows/ci.yml:38]: Oh, I see, for installing curl and serverless. I couldn't find anything definitive, but it seems to run ok on staging without sudo.

**Pull request merged**